layer/src/conv.rs

```diff
@@ -75,21 +81,8 @@ impl Conv1D {
         let kernel_size = weights_shape[2];
         let dilated_kernel_size = dilation * (kernel_size - 1) + 1;
 
-        if groups > channel_out_size {
-            return Err(ConvError::Config(format!(
-                "given groups={}, expected weight to be at least {} at dimension 0, but got weight of size {:?} instead",
-                groups,
-                groups,
-                weights_shape,
-            )));
-        }
```

Comment:
is it ok to remove this?

Reply:
yes, it's already implied: both `groups` and `channel_out_size` are guaranteed to be non-zero, hence any `channel_out_size < groups` will cause `channel_out_size % groups == channel_out_size != 0` wich is already checked below.